SPECS/cri-o/cri-o.spec

```diff
@@ -0,0 +1,1711 @@
+#
+# spec file for package cri-o
+#
+# Copyright (c) 2018 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+%define debug_package %{nil}
+%define project github.com/cri-o/cri-o
+#Compat macro for new _fillupdir macro introduced in Nov 2017
+%if ! %{defined _fillupdir}
+  %define _fillupdir %{_var}/adm/fillup-templates
+%endif
+Summary:        OCI-based implementation of Kubernetes Container Runtime Interface
+# Define macros for further referenced sources
+Name:           cri-o
+Version:        1.21.2
+Release:        2%{?dist}
+License:        Apache-2.0
```

Comment:
Could you replace this with one of the short names from [this table](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses)? I believe it should be "ASL 2.0", but I'd double-check with the project's license info.